index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.css">
+    <title>YouClone | Player: How to build a YouTube Clone</title>
+
+</head>
+<body>
```

Comment:
It'd be good if you add `<main>` tags for the dominant content of your body moving on

Reply:
Thanks for your feedback! We are going to use this from now on! :)